extensions/cornerstone/src/commandsModule.ts

```diff
@@ -881,8 +886,29 @@ function commandsModule({
         });
       }
     },
-    rotateViewport: ({ rotation }) => {
-      const enabledElement = _getActiveViewportEnabledElement();
+    /**
+     * Rotates the viewport by the given rotation amount.
+     * @param rotation - Degrees clockwise to rotate the viewport by.
+     * @param viewportId - The ID of the viewport to rotate.
+     * @param rotationMode - The mode to use for the rotation. 'apply' will add the rotation to the current rotation, 'set' will set the rotation to the given rotation.
+     */
+    rotateViewport: ({
+      rotation,
+      viewportId = 'currentlyActive',
+      rotationMode = 'apply',
+    }: {
```

Comment:
You can make `viewportId` optional. If it's not defined, we can use the active viewport. We already use this pattern throughout the code: when `viewportId` isn't defined, it means we should use the active one. I prefer this to a new magic string `currentlyActive`